content/applications/general/payment_acquirers/paypal.rst

```diff
@@ -82,59 +96,48 @@ We advise you to not prompt customers to log in with a Paypal account when they
-guaranteed since IPN resends a confirmation until your site acknowledges receipt.
-
-| To activate IPN, get back to *Website payments* menu and click *Update* in *Instant Payment
-  Notification*.
-| The *Notification URL* to set is your domain name + “payment/paypal/ipn” (e.g.
-  `https://yourcompany.odoo.com/payment/paypal/ipn`).
-
 Payment Messages Format
 -----------------------
 
-If you use accented characters (or anything else than basic Latin characters) for your customer
-names, addresses... you MUST configure the encoding format of the payment request sent by Odoo to
-Paypal.
-
-.. danger::
-   If you don't configure this setting, some transactions fail without notice.
+Suppose you use accented characters (or anything else than primary Latin characters) for your
+customer names or addresses. In that case, you **must** configure the encoding format of the payment
+request sent by Odoo to Paypal. Otherwise, some transactions fail without notice.
 
-To do so, open:
-
-- `this page for a test account <https://sandbox.paypal.com/cgi-bin/customerprofileweb?cmd=_profile-language-encoding>`_
-- `this page for a production account <https://www.paypal.com/cgi-bin/customerprofileweb?cmd=_profile-language-encoding>`_
-
-Then, click *More Options* and set the two default encoding formats as **UTF-8**.
+To do so, go to `your production account <https://www.paypal.com/cgi-bin/
+customerprofileweb?cmd=_profile-language-encoding>`_. Then, click *More Options* and set the two
+default encoding formats as **UTF-8**.
 
 Your Paypal account is ready!
 
 .. tip::
-   For Encrypted Website Payments & EWP_SETTINGS error, please check the `Paypal documentation
-   <https://developer.paypal.com/docs/classic/paypal-payments-standard/integration-guide/
-   encryptedwebpayments#encrypted-website-payments-ewp>`_.
+   * For Encrypted Website Payments & EWP_SETTINGS error, please check the `Paypal documentation
+     <https://developer.paypal.com/docs/classic/paypal-payments-standard/integration-guide/
+     encryptedwebpayments#encrypted-website-payments-ewp>`_.
+  * Configure your :ref:`Paypal Sandbox account <paypal/testing>`., then follow this
+    `link <https://sandbox.paypal.com/cgi-bin/customerprofileweb?cmd=_profile-language-encoding>`_
+    to configure the encoding format in a test environment.
+
+.. _paypal/testing:
 
 Test environment
 ================
 
-You can test the entire payment flow in Odoo thanks to Paypal Sandbox accounts.
-
-Log in to `Paypal Developer Site <https://developer.paypal.com/>`__ with your Paypal credentials.
+Configuration
+-------------
 
-This will create two sandbox accounts:
+Thanks to Paypal Sandbox accounts, you can test the entire payment flow in Odoo.
 
--  A business account (to use as merchant, e.g. `pp.merch01-facilitator@example.com <mailto:pp.merch01-facilitator@example.com>`__).
+Log into the `Paypal Developer Site <https://developer.paypal.com/>`__ using your Paypal
```

Comment:
Since you change this line, take the opportunity to remove the extra underscore. (it works with 2 but it's not ideal)
```suggestion
Log into the `Paypal Developer Site <https://developer.paypal.com/>`_ using your Paypal
```